cmd/neofs-cli/modules/object.go

```diff
@@ -1,24 +1,125 @@
+		Long:  `Operations with Objects`,
+		Run: func(cmd *cobra.Command, args []string) {
+			cmd.Println("object called")
+		},
+	}
+
+	objectPutCmd = &cobra.Command{
+		Use:   "put",
+		Short: "Put object to NeoFS",
+		Long:  "Put object to NeoFS",
+		RunE:  putObject,
+	}
+
+	objectGetCmd = &cobra.Command{
+		Use:   "get",
+		Short: "Get object from NeoFS",
+		Long:  "Get object from NeoFS",
+		RunE:  getObject,
+	}
+
+	objectDelCmd = &cobra.Command{
+		Use:     "delete",
+		Aliases: []string{"del"},
+		Short:   "Delete object from NeoFS",
+		Long:    "Delete object from NeoFS",
+		RunE:    deleteObject,
+	}
+
+	objectHeadCmd = &cobra.Command{
+		Use:   "head",
+		Short: "Get object header",
+		Long:  "Get object header",
+		RunE:  getObjectHeader,
+	}
+
+	objectHashCmd = &cobra.Command{
+		Use:   "hash",
+		Short: "Get object hash",
+		Long:  "Get object hash",
+		RunE:  getObjectHash,
+	}
+)
+
+const (
+	hashSha256 = "sha256"
+	hashTz     = "tz"
+	rangeSep   = ":"
+)
 
 func init() {
 	rootCmd.AddCommand(objectCmd)
 
+	objectCmd.AddCommand(objectPutCmd)
+	objectPutCmd.Flags().String("file", "", "File with object payload")
+	_ = objectPutCmd.MarkFlagFilename("file")
+	_ = objectPutCmd.MarkFlagRequired("file")
+	objectPutCmd.Flags().String("cid", "", "Container ID")
+	_ = objectPutCmd.MarkFlagRequired("cid")
+	objectPutCmd.Flags().StringSlice("attributes", []string{}, "User attributes in form of Key1=Value1,Key2=Value2")
+
+	objectCmd.AddCommand(objectDelCmd)
+	objectDelCmd.Flags().String("cid", "", "Container ID")
+	_ = objectDelCmd.MarkFlagRequired("cid")
+	objectDelCmd.Flags().String("oid", "", "Object ID")
+	_ = objectDelCmd.MarkFlagRequired("oid")
+
+	objectCmd.AddCommand(objectGetCmd)
+	objectGetCmd.Flags().String("file", "", "File to write object payload to. Default: stdout.")
+	objectGetCmd.Flags().String("header", "", "File to write header to. Default: stdout.")
+	objectGetCmd.Flags().String("cid", "", "Container ID")
+	_ = objectGetCmd.MarkFlagRequired("cid")
+	objectGetCmd.Flags().String("oid", "", "Object ID")
+	_ = objectGetCmd.MarkFlagRequired("oid")
+
+	objectCmd.AddCommand(objectHeadCmd)
+	objectHeadCmd.Flags().String("file", "", "File to write header to. Default: stdout.")
+	objectHeadCmd.Flags().String("cid", "", "Container ID")
+	_ = objectHeadCmd.MarkFlagRequired("cid")
+	objectHeadCmd.Flags().String("oid", "", "Object ID")
+	_ = objectHeadCmd.MarkFlagRequired("oid")
+	objectHeadCmd.Flags().Bool("json", false, "Marshal output in JSON")
+	objectHeadCmd.Flags().Bool("proto", false, "Marshal output in Protobuf")
+
+	objectCmd.AddCommand(objectHashCmd)
+	objectHashCmd.Flags().String("cid", "", "Container ID")
+	_ = objectHashCmd.MarkFlagRequired("cid")
+	objectHashCmd.Flags().String("oid", "", "Object ID")
+	_ = objectHashCmd.MarkFlagRequired("oid")
+	objectHashCmd.Flags().StringSlice("range", []string{}, "Range to take hash from.")
+	objectHashCmd.Flags().String("type", hashSha256, "Hash type. Either 'sha256' (default) or 'tz'")
```

Comment:
Default value specified automatically by Cobra, so you don't have to comment about this. 
Now help line looks
```
      --type string     Hash type. Either 'sha256' (default) or 'tz' (default "sha256")
```